.github/workflows/docker.yaml

```diff
@@ -0,0 +1,70 @@
+name: Publish images to dockerhub
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "main"
```

Comment:
Could you make sure to build for the `dev` branch as well? It's the staging branch for this repo.

Reply:
> Could you make sure to build for the `dev` branch as well? It's the staging branch for this repo.

done! would need changes merged/picked into dev branch as well.